src/js/apps/patients/schedule/schedule_app.js

```diff
@@ -94,6 +94,17 @@ export default App.extend({
           .done(() => {
             Radio.request('alert', 'show:success', renderTemplate(BulkEditActionsSuccessTemplate, { itemCount: this.selected.length }));
             app.stop();
+
+            if (saveData.due_date) {
+              const isDueDateChanged = this.selected.some(action => action.changed.due_date);
+
+              if (isDueDateChanged) {
+                this.getState().clearSelected();
```

Comment:
Does this line need to happen after `this.selected.some`?  I suspect not, in which case, this and L108 could just move up to L97

Reply:
Yeah, that's true. I thought the `some` was going to be multi-lined, so I separated them out. I'll consolidate.

Reply:
oh yeah the `if`s could be combined, but I was looking at `this.getState().clearSelected();` called here and then below.  it can probably happen before the `if`